conformance/utils/suite/conformance.go

```diff
@@ -44,13 +45,19 @@ func (test *ConformanceTest) Run(t *testing.T, suite *ConformanceTestSuite) {
 		t.Parallel()
 	}
 
+	var featuresInfo string
 	// Test against features if the user hasn't focused on a single test
 	if suite.RunTest == "" {
 		// Check that all features exercised by the test have been opted into by
 		// the suite.
-		for _, feature := range test.Features {
-			if !suite.SupportedFeatures.Has(feature) {
-				t.Skipf("Skipping %s: suite does not support %s", test.ShortName, feature)
+		for i, featureName := range test.Features {
+			if !suite.SupportedFeatures.Has(featureName) {
+				t.Skipf("Skipping %s: suite does not support %s", test.ShortName, featureName)
+			}
+			feature := features.GetFeature(featureName)
+			featuresInfo = fmt.Sprintf("%s%s-%s, ", featuresInfo, feature.Name, feature.Status)
+			if i < len(featureName)-1 {
+				featuresInfo += ", "
 			}
```

Comment:
nit: You already have this comma covered in L58.

Reply:
Good catch! I think the goal of having this in the the if block is to avoid a trailing comma, but if that's the case, we should remove the comma from the Sprintf on line 58, or update this if to remove a trailing comma from the string for the last item in the list.

Reply:
yep, you are right. Removed 👍